src/config/config.yml

```diff
@@ -48,4 +53,7 @@ trailerIdInitialValue:
   trailerLetter: C
   sequenceNumber: 530000
   testNumberKey: 2
+systemNumberInitialValue:
+  systemNumber: "10000000"
+  testNumberKey: 3
```

Comment:
this `testNumberKey` is not used. for all three `NUMBER_KEY ` enum is being used. We can remove this attribute.

Reply:
I don't think I get what you're saying. That is the hashKey
        KeySchema:
        - AttributeName: testNumberKey
          KeyType: HASH

Reply:
Got it